css/properties/aspect-ratio.json

```diff
@@ -28,6 +28,9 @@
               "version_added": "88"
             },
             "firefox": [
+              {
+                "version_added": "89"
+              },
```

Comment:
The [intent to ship](https://bugzilla.mozilla.org/show_bug.cgi?id=1672073#c2) says enabled for all platforms. Would you mind adding this support statement to Firefox for Android, below?